### PR TITLE
Sample for configuring SSL enabled worker

### PR DESCRIPTION
**Pull request by vitarb**
Shows how to setup SSL enabled worker.
Also adds a dependency on recently added testing library.

### CODE REVIEW

File: src/main/java/io/temporal/samples/ssl/SslEnabledWorker.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    // backend.
+    // client that can be used to start and signal workflows.
+    WorkflowClient client =
+        WorkflowClient.newInstance(
+            service, WorkflowClientOptions.newBuilder().setNamespace(namespace).build());
+    // worker factory that can be used to create workers for specific task queues
+    WorkerFactory factory = WorkerFactory.newInstance(client);
+    // Worker that listens on a task queue and hosts both workflow and activity implementations.
+    Worker worker = factory.newWorker(TASK_QUEUE);
+    // TODO now register your workflow types and activity implementations.
```
Comment (Sushisource) [inline src/main/java/io/temporal/samples/ssl/SslEnabledWorker.java:69]: This TODO, is it meant for the person reading the sample?

If so maybe remove the TODO and say "Now, here is where you would register... as in the below two lines"

**Review by Sushisource — APPROVED** [review]
LGTM small wording change

File: src/main/java/io/temporal/samples/ssl/SslEnabledWorker.java (hunk 1 above)
Comment (vitarb) [reply src/main/java/io/temporal/samples/ssl/SslEnabledWorker.java:69]: I wanted to make it more obvious that it's a required part, and didn't want to copy-paste some other sample, as it's not focused on the specific behavior. Having TODO would highlight the line in most people's IDE, making it more clear.

Comment (Sushisource) [reply src/main/java/io/temporal/samples/ssl/SslEnabledWorker.java:69]: Ah, sure that makes sense. Might keep the TODO and the wording upgrade so it's a bit more clear it's a call to action.

**Pull request merged**